pkg/workceptor/kubernetes.go

```diff
@@ -690,39 +724,8 @@ func shouldUseReconnect(kw *kubeUnit) bool {
 		return false
 	}
 
-	semver, err := version.ParseSemantic(serverVerInfo.String())
-	if err != nil {
-		logger.Warning("could parse Kubernetes server version %s, will not use reconnect support", serverVerInfo.String())
-
-		return false
-	}
-
-	// ignore pre-release in version comparison
-	semver = semver.WithPreRelease("")
```

Comment:
I'm inclined to remove this -- seems too risky to ignore the pre-release tags. I know there was at least one k8s cluster out there that doesn't follow version semantics, so reconnect should be possible but auto detect says no.

that is fine, they can enable it through the ENV. However, incorrectly enabling reconnect support when in fact the k8s cluster doesn't support it is really bad. I'd rather just have them experience the 4 hour log problem in the meantime.

Reply:
Main problem is that the fix is not in k8s API server code and it's in kubelet code so we actually needed to read node version not k8s apiserver version 🤦‍♂️

Reply:
and i think we should still put the pre-release stripping in

Reply:
added back